tests/arch/arm/arm_interrupt/src/arm_interrupt.c

```diff
@@ -440,7 +438,7 @@ void test_arm_user_interrupt(void)
 #endif
 }
 #else
-void test_arm_user_interrupt(void)
+ZTEST(arm_interrupt, test_arm_user_interrupt)
```

Comment:
Discussed offline, but adding here for tracking. Should be `ZTEST_USER`

Reply:
So then maybe this was a bug the whole time?

The original test suite did not test this as a user test.

Reply:
Fixed.